cmd/lotus-shed/verifreg.go

```diff
@@ -24,9 +24,10 @@ import (
 )
 
 var verifRegCmd = &cli.Command{
-	Name:  "verifreg",
-	Usage: "Interact with the verified registry actor",
-	Flags: []cli.Flag{},
+	Name:   "verifreg",
+	Usage:  "Interact with the verified registry actor",
+	Flags:  []cli.Flag{},
+	Hidden: true,
```

Comment:
Note this also hides our "use root key to make a verifier" util, but that should be fine...could alternatively hide all the subcommands except that one.

Reply:
I'd do this per-command (and print the 'this is depcecated' warnings)